test/action/dependency.test.ts

```diff
@@ -82,16 +82,15 @@ describe('package.json', function () {
         });
 
         if (depName === 'snappy') {
-          itInNodeProcess(
-            'getSnappy returns rejected import',
-            async function ({ expect, mongodb }) {
-              const snappyImport = mongodb.getSnappy();
-              expect(snappyImport).to.have.nested.property(
-                'kModuleError.name',
-                'MongoMissingDependencyError'
-              );
-            }
-          );
+          itInNodeProcess('getSnappy returns rejected import', async function ({ expect }) {
+            // @ts-expect-error: import from the inside forked process
+            const { getSnappy } = await import('./src/deps.ts');
```

Comment:
This test (and the one below) forks node process and setup clean project to test import dependencies. Before this change inside this process the entire `mongodb.ts` was imported and passed as object inside `run` function along with `chai.expect`. In the new approach we let test import `getSnappy` (and potentially any other functions or entities) directly. Only `{expect}` therefore will be provided.